app/utils/print/sections/AnswerRowConverter.scala

```diff
@@ -193,6 +193,21 @@ class AnswerRowConverter @Inject()(checkAnswersFormatters: CheckAnswersFormatter
     }
   }
 
+  def numberOfBeneficiariesQuestion[T](query: Gettable[T],
```

Comment:
Also minor but with this method being specific to number of beneficiaries the generic type `[T]` is not necessary. Can instead just make it `Gettable[HowManyBeneficiaries]`